app-tasks/rf/src/rf/utils/io.py

```diff
@@ -5,6 +5,7 @@
 import shutil
 import subprocess
 import tempfile
+import time
```

Comment:
Doesn't look like this import is being used

Reply:
Well it _was_, _once_, a minute ago before I had to move things to resolve a surprising circular import

...

...

removing 🤦🏻‍♂️

Reply:
20a8dfb